content/strategy-goals/strategy/code-intelligence/index.md

```diff
@@ -79,22 +79,21 @@ Top feedback patterns we're hearing from customers and prospects across the boar
 
 ### Themes
 
-**Unlock dependency navigation**
+**Make code navigation feel fast and reliable**
 
-We believe this is emerging as an additional killer feature. Most developers think in terms of package and dependency versions rather than focusing on external repositories. Supporting precise dependency navigation elevates the code navigation experience to a new level of cross-project analysis. Dependency navigation coupled with cross-repository takes the IDE model and scales it to encompass the whole cognitive space of a company.
+We receive a significant amount of feedback around papercuts and potential improvements to our navigation experience. We are commited to making our navigation feel fast and reliable to our users. Our current focus is in solving long-standing navigation issues while also adding quality of life improvements with the aim of strenghthening the core workflow that the majority of users interact with daily.
```

Comment:
What are the top things we're going to tackle here? Are there any actionable metrics around speed, if not, will we add those?